deploy/04-rbac-rolebinding.yaml

```diff
@@ -0,0 +1,27 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: default-account-console-operator
+  namespace: openshift-console
+subjects:
+- kind: ServiceAccount
+  # name: console-operator
```

Comment:
the `sdk` struggles to do things if I change this, looking into it as a follow-on as well.